docs/source/processors/panel.rst

```diff
@@ -17,10 +20,15 @@ Required Tag Parameters
 
 - ``title`` - Text to display as the panel's title.
```

Comment:
This feels wrong to still include here - title is no longer a parameter of the tag, and you've already outlined above how the title and subtitle are extracted